src/Components/Notes/NoteCardCreate.jsx

```diff
@@ -0,0 +1,190 @@
+import React, {useState} from 'react'
+import {useAuth0} from '@auth0/auth0-react'
+import Avatar from '@mui/material/Avatar'
+import Box from '@mui/material/Box'
+import CardContent from '@mui/material/CardContent'
+import CardHeader from '@mui/material/CardHeader'
+import InputBase from '@mui/material/InputBase'
+import Paper from '@mui/material/Paper'
+import useTheme from '@mui/styles/useTheme'
+import {TooltipIconButton} from '../Buttons'
+import useStore from '../../store/useStore'
+import {postIssue} from '../../utils/GitHub'
+import Submit from '../../assets/icons/Submit.svg'
+
+
+/**
+ * Note card
+ *
+ * @param {number} id note id
+ * @param {number} index note index
+ * @param {string} username username of the note author
+ * @param {string} title note title
+ * @param {string} avatarUrl user avatarUrl
+ * @param {string} body note body
+ * @param {string} date note date
+ * @param {number} numberOfComments number of replies to the note - referred to as comments in GH
+ * @param {boolean} expandedImage governs the size of the image, small proportions on mobile to start
+ * @param {boolean} isComment Comments/replies are formatted differently
+ * @return {object} React component
+ */
+export default function NoteCardCreate({
+  username = '',
+  avatarUrl = '',
+}) {
+  const repository = useStore((state) => state.repository)
+  const createNoteOff = useStore((state) => state.createNoteOff)
+  const [title, setTitle] = useState('')
+  const [body, setBody] = useState('')
+  const {user, isAuthenticated} = useAuth0()
+  const accessToken = useStore((state) => state.accessToken)
+  const createdNotes = useStore((state) => state.createdNotes)
+  const setCreatedNotes = useStore((state) => state.setCreatedNotes)
+  const theme = useTheme()
+
+
+  /**
+   * create issue takes in the title and body of the note
+   *
+   */
+  function createNote() {
+    // TODO:Oleg noteIndex is used for note number :: need to introduce internal index system
```

Comment:
Pls use `TODO(username)` format.. goal here is to have a standard way to search these later